### PR TITLE
Update storage-configuration.md

### PR DESCRIPTION
**Pull request by ujpat**

### CODE REVIEW

**Comment by Court72** [comment]
@bluefooted

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/virtual-machines/windows/storage-configuration.md [hunk 1]
```diff
@@ -272,6 +272,10 @@ There's no option to remove the unallocated space from a disk that belongs to a
 
 Review [Troubleshoot a full transaction log](/sql/relational-databases/logs/troubleshoot-a-full-transaction-log-sql-server-error-9002) if your log becomes full. 
 
+### Unable to Enable Storage Configuration from the Azure Portal
+
+When TCP/IP is disabled, the storage configuration will fail to set up. Review SQL Server configuration manager to check TCP/IP Settings.
```
Comment (MashaMSFT) [inline azure-sql/virtual-machines/windows/storage-configuration.md]:
```suggestion
Storage configuration for the SQL virtual machines resource in the Azure portal isn't available in the following scenarios: 
- Virtual machines with self-installed SQL Server instances. Currently, only SQL Server VM images from Azure marketplace are supported.  
- SQL Server VMs using Premium SSDv2. Currently, only SQL Server VMs with Premium SSD are supported. 
- When TCP/IP is disabled in SQL Server Configuration Manager. 
```

File: azure-sql/virtual-machines/windows/storage-configuration.md [hunk 2]
```diff
@@ -272,6 +272,10 @@ There's no option to remove the unallocated space from a disk that belongs to a
 
 Review [Troubleshoot a full transaction log](/sql/relational-databases/logs/troubleshoot-a-full-transaction-log-sql-server-error-9002) if your log becomes full. 
 
+### Unable to Enable Storage Configuration from the Azure Portal
```
Comment (MashaMSFT) [inline azure-sql/virtual-machines/windows/storage-configuration.md]:
```suggestion
### The Storage pane for the SQL virtual machines resource is unavailable in the Azure Portal
```

**Comment by MashaMSFT** [comment]
Pulled this change into the private repo

**Pull request closed without merging**